### ISSUE TITLE
Better error message needed for `convox debug`

### ISSUE THREAD

**Issue by mattmanning** [convox/cli#88]
The error message returned when debug is run for an app that doesn't exist doesn't make any sense:

```
$ convox debug
ERROR: InvalidParameterException: Services cannot be empty.
	status code: 400, request id: []
```

**Comment by steinnes** [comment]
+1 on this one.  I tried grepping for this error in this project codebase, but I couldn't find it so I assume this is an error returned by AWS, or some other part of convox.

**Comment by ddollar** [comment]
Closed in favor or convox/kernel#119
